CHANGELOG.md

```diff
@@ -1,3 +1,59 @@
+
+## Breaking Changes - Action Required
+> A `--full-refresh` is **required** after upgrading to prevent errors caused by naming and materialization changes. Additionally, downstream queries **must** be updated to reflect new model and column names.
+
+- The materialization of all `stg_*` staging models has been updated from `table` to `view`.
+  - Previously `stg_*_tmp` models were views while the non-`*_tmp` versions were tables. Now all are views to eliminate redundant data storage.
+
+- **Source Table Transition:**
+  - The `CONNECTOR` source table is deprecated and replaced by `CONNECTION`. During a brief transition period, both tables will be identical, but `CONNECTOR` will stop receiving data and be removed at a later time.
+    - This change clarifies the distinction: **Connectors** facilitate the creation of **connections** between sources and destinations.
+  - The `CONNECTION` table is now the default source
+    - **For Quickstart users:** The `CONNECTOR` will automatically be used if `CONNECTION` is not yet available.
+    - **For dbt Core users:** Users without the `CONNECTION` source can continue using `CONNECTOR` by adding the following variable to your root `dbt_project.yml` file:
+      ```yml
+      vars:
+          fivetran_platform_using_connection: false # default: true
+      ```
+    - For more details, refer to the [README](https://github.com/fivetran/dbt_fivetran_log/blob/main/README.md#leveraging-connection-vs-connector).
+
+- Model Renames:
+  - `fivetran_platform__connector_status` → `fivetran_platform__connection_status`
+  - `fivetran_platform__connector_daily_events` → `fivetran_platform__connection_daily_events`
+  - `fivetran_platform__usage_mar_destination_history` → `fivetran_platform__usage_history`
+  - `stg_fivetran_platform__connector` → `stg_fivetran_platform__connection`
+  - `stg_fivetran_platform__connector_tmp` → `stg_fivetran_platform__connection_tmp`
+> **NOTE**: Ensure any downstream queries are updated to reflect the new model names.
+
+- Column Renames:
+  - Renamed `connector_id` to `connection_id` and `connector_name` to `connection_name` in the following models:
+    - `fivetran_platform__connection_status`
+      - Also renamed `connector_health` to `connection_health`
+    - `fivetran_platform__mar_table_history`
+    - `fivetran_platform__connection_daily_events`
+    - `fivetran_platform__audit_table`
+    - `fivetran_platform__audit_user_activity`
+    - `fivetran_platform__schema_changelog`
+    - `stg_fivetran_platform__connection`
+    - `stg_fivetran_platform__log`
+    - `stg_fivetran_platform__incremental_mar`
+  - These renames 
+> **NOTE**: Ensure any downstream queries are updated to reflect the new column names.
+
+- New Columns:
+  - As part of the `CONNECTION` updates, the following columns have been added alongside their `connector_*` equivalents:  
+    - INCREMENTAL_MAR: `connection_id` and `connection_name`  
+    - LOG: `connection_id`
```

Comment:
Looks great, but can you move this to above the `Model Renames` section? So all the source updates are somewhat together